### ISSUE TITLE
Make fmpz_poly_q available in Nemo

### ISSUE THREAD

**Issue by heiderich** [Nemocas/Nemo.jl#574]
It is possible to make `fmpz_poly_q` from FLINT available in Nemo?

**Comment by wbhart** [comment]
It would be possible, but I don't know anyone who has time to do it though.

It's also not clear what the interface should be, as it is just another rational function module I guess. I'm also not sure how much faster it would be, if at all.

**Comment by heiderich** [comment]
Is there already a "rational function module"? I must have missed it. Could you please tell me where to find it?

**Comment by heiderich** [comment]
Oh, I think I just found it in AbstractAlgebra:

https://nemocas.github.io/AbstractAlgebra.jl/latest/fraction_fields/
https://nemocas.github.io/AbstractAlgebra.jl/latest/fraction/

I was just searching for "quotient field". Maybe it would be wroth including this string somewhere in the documentation, so that it can be found easily.

**Comment by wbhart** [comment]
That's a good idea. We've avoided using the term because people generally expect much more from rational functions than we provide. The basic arithmetic only is available at present, no extensions.

By the way, we recently did a lot of work to radically speed up the rational functions, so I think you will find it performs really well. I doubt we could beat it in Flint. Also, you do not need to call Generic.FractionField or anything like that. It will work from Nemo itself by just calling FractionField.
